src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -71,17 +71,14 @@ def tensor2vid(video: torch.Tensor, processor: "VaeImageProcessor", output_type:
     for batch_idx in range(batch_size):
         batch_vid = video[batch_idx].permute(1, 0, 2, 3)
         batch_output = processor.postprocess(batch_vid, output_type)
-
         outputs.append(batch_output)
 
     if output_type == "np":
         outputs = np.stack(outputs)
-
-    elif output_type == "pt":
+    elif output_type == "pt" or output_type == "latent":
```

Comment:
how would the `latent` type output be used for animate diff?

Reply:
We already support latents as input to `__call__` right? Animate vid2vid, pia, controlnet, etc. can use latents generated by other pipelines.

The main focus of this PR is to handle return_dict correctly because all pipelines here would return a class object when return_dict=False and output_type=latent , but the expected is a tuple. Since pt and latent are essentially the same output type, like the other pipelines, the extra if-statement doesn't make sense in post processing.

Reply:
I'm AFK for the next few hours so please hold on if this gets approved for merge 😔 The return_dict parameter is not properly implemented in the community pipelines of animatediff. This is causing some issues on a downstream repo, so making the behaviour consistent and fixing across all pipes here would be awesome.